pkg/network/sriov/sriov_test.go

```diff
@@ -0,0 +1,313 @@
+/*
+* This file is part of the KubeVirt project
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+* Copyright 2022 Red Hat, Inc.
+*
+ */
+
+package sriov_test
+
+import (
+	"kubevirt.io/kubevirt/pkg/network/sriov"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	virtv1 "kubevirt.io/api/core/v1"
+)
+
+var _ = Describe("SRIOV", func() {
+	nonValidNetworkStatusAnnotationValue := ``
+	networkStatusWithOneNetwork := `
```

Comment:
nit: I suggest naming this variable consistently with the other two: `networkStatusWithOneSRIOVNetwork`

Reply:
Done